src/views/SearchOverview/PoolDropdown.tsx

```diff
@@ -23,7 +25,8 @@ export const PoolDropdown = ({ pool, setPool }: DropdownProps) => {
       options={pools}
       onSelect={(e) => {
         setPool(e.value)
+        setQueryString({ ...query, s: e.value })
```

Comment:
the setter is appending, so spreading query is unnecessary.
The key for the parameter should be discussed, I like explicit naming which means I've used from=, id= source= etc. It gives longer but clearer urls.

Either way, it's best if we're consistant

Reply:
should be fixed in new commit